src/index.js

```diff
@@ -71,9 +71,18 @@ window.newfoldEmbeddedHelp.toggleNFDLaunchedEmbeddedHelp = () => {
 //For rendering embedded help in Add, edit and View Pages
 const HelpCenterPluginSidebar = () => {
   const [helpEnabled, setHelpEnabled] = useState(false);
-  CapabilityAPI.getHelpCenterCapability().then((response) => {
-    setHelpEnabled(response);
-  });
+  const getHelpStatus = async () => {
```

Comment:
I remember changing it to then, response because useEffect doesn't always work as the page is not essentially re-rendered but just loaded back on, has this been tested ?

Reply:
I have reverted the changes as it wasn't re-rendering but [helpEnabled, setHelpEnabled] are defined in the parent as well as the child component. There are chances of state duplication and the state might not be in sync for example the gutenberg page crashes on local when I am mocking the Capability check with response 'true' as a promise.